examples/demos/example21-cell-selection-events.js

```diff
@@ -30,20 +30,15 @@ export default class extends React.Component {
     return this._rows[index];
   };
 
-  onCellSelected = ({ rowIdx, idx }) => {
+  onSelectedCellChange = ({ rowIdx, idx }) => {
```

Comment:
So there will never be a deselection api, right? What I am trying to say is to not change the example behavior in this case, so perhaps we can add `selectedRowIdx` and `selectedColIdx` in this example `state`, and the next time when the `onSelectedCellChange` is called, the `this.state.selectedRowIdx` and `this.state.selectedColIdx` are the deselected selection and the new selection is `rowIdx, idx`.

Reply:
I think it is extra work and probably we can skip it :)

Reply:
I believe it's better to keep it so that people won't question where is the `onDeselectedCellChange`. Or at least put a comment about how to use it like `onDeselectedCellChange`

Reply:
I have updated the example